EIPS/eip-6239.md

```diff
@@ -0,0 +1,323 @@
+
+Semantic SBTs address this by storing social metadata, attestations, and access permissions on-chain to bootstrap the social identity layer and a linked data layer natively on Ethereum, and bring semantic meanings to the tons of bits of on-chain data.
+
+### Connectedness
+
+Semantic SBTs store social data as RDF triples in the Subject-Predicate-Object format, making it easy to create relationships between accounts and attributes.  RDF is a standard for data interchange used to represent highly interconnected data. Representing data in RDF triples makes it simpler for automated systems to identify, clarify, and connect information.
+
+### Linked Data
+
+Semantic SBTs allow the huge amount of social data on-chain to be available in a standard format (RDF) and be reachable and manageable. The interrelated datasets on-chain can create the linked data layer that allows social data to be mixed, exposed, and shared across different applications, providing a convenient, cheap, and reliable way to retrieve data, regardless of the number of users.
+
+### Social Identity
+
+Semantic SBTs allow people to publish or attest their own identity-related data in a bottom-up and decentralized way, without reliance on any centralized intermediaries while setting every party free. The data is fragmentary in each Semantic SBT and socially interrelated. RDF triples enable various community detection algorithms to be built on top.
+
+This proposal outlines the semantic data modeling of SBTs that allows implementers to model the social relations among Semantic SBTs, especially in the social sector.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+- The token **MUST** implement the following interfaces:
+    1. [ERC-165](./eip-165.md)’s `ERC165` (`0x01ffc9a7`)
+    1. [ERC-721](./eip-721.md)’s `ERC721` (`0x80ac58cd`)
+    1. [ERC-721](./eip-721.md)’s `ERC721Metadata` (`0x5b5e139f`)
+    1. [ERC-5192](./eip-5192.md)’s `ERC5192` (`0xb45a3c0e`)
+
+### RDF Statement
+
+RDF statements come in various formats, we have selected the six most commonly used formats: `nt(N-Triples)`,`ttl(Turtle)`,`rdf(RDF/XML)`,`rj(RDF/JSON)`,`nq(N-Quads)` and `trig(TriG)`.
+
+The complete format of an RDF statement：
+
+```text
+rdfStatements = {[format]}<statements>
+```
+
+{}：OPTIONAL
+
+<>: MUST
+
+format: nt/ttl/rdf/rj/nq/trig
+
+When no format is selected: statements = [ttl]statements
+
+- `nt(n-triples)`
+
+[`nt`](http://www.w3.org/ns/formats/N-Triples) uses space to separate the subject, predicate, object of a triple, and a period . to indicate the end of a triple.
```

Comment:
We currently do not allow links to the W3 standards organization. Once https://github.com/ethereum/EIPs/pull/6691 is merged, the bots will be updated to allow these links.

In the meantime, you could remove these links, or wait for that pull request to be merged.